functions/Copy-DbaDatabase.ps1

```diff
@@ -995,10 +995,17 @@ function Copy-DbaDatabase {
 					}
 
 					$dbFinish = Get-Date
-
 					if ($NoRecovery -eq $false) {
+						$dbOwner = $sourceServer.Databases[$dbName].Owner
```

Comment:
I copied the logic from below but I think it's an error. I'm going to update the code here and there to:
- get the owner from the source
- if it's null (some weird error may happen) OR the owner from source doesn't exist in destination, THEN fetch the sa from the destination